src/app/components/applications/SSV/RegisterValidatorHome/components/ImportFile/flows/KeyStore.tsx

```diff
@@ -59,17 +58,17 @@ const KeyStoreFlow = () => {
     }
   };
 
-  const isDeposited = async (): Promise<boolean> => {
-    const beaconChaValidatorUrl = `${getBeaconChainLink()}/api/v1/validator/${validatorStore.keyStorePublicKey}/deposits`;
-    try {
-      const response: any = (await axios.get(beaconChaValidatorUrl, { timeout: 5000 })).data;
-      const conditionalDataExtraction = Array.isArray(response.data) ? response.data[0] : response.data;
-      return !(response.status === 'OK' && conditionalDataExtraction?.valid_signature === undefined);
-    } catch (e: any) {
-      console.log(e.message);
-      return true;
-    }
-  };
+  // const isDeposited = async (): Promise<boolean> => {
```

Comment:
remove it

Reply:
we have ticket in backlog to start using this function :)